integration-tests/features/steps/stack_analyses_v2.py

```diff
@@ -234,6 +271,157 @@ def verify_vulnerability_count(context, component, vulnerability, expected):
     assert is_component_present, 'Component not present'
 
 
+def verify_dependency_count(context, component, expected):
+    """Check dependency count for given component."""
+    json_data = get_json_data(context)
+    check_attribute_presence(json_data, 'analyzed_dependencies')
+
+    is_component_present = False
+    for dep in json_data['analyzed_dependencies']:
+        if dep['name'] == component:
+            is_component_present = True
+            dependencies = dep['dependencies']
+            print(dep)
+            actual = len(dependencies)
+            assert actual == expected, \
+                'Component {} have {} deps but expected {}'.format(component, actual,
+                                                                   expected)
+    assert is_component_present, 'Component not present'
```

Comment:
Can you have correct message, add a component name so that the error is clearer

Reply:
okay